library/gradle

```diff
@@ -1,57 +1,57 @@
 Maintainers: Keegan Witt <keeganwitt@gmail.com> (@keeganwitt)
 GitRepo: https://github.com/keeganwitt/docker-gradle.git
-GitCommit: ef27237d21e823cc0e075fff946322b5e551e8c4
+GitCommit: 605a08fb025ad34e8e9d5d2391427886e64fc9f7
 
 
 # Hotspot
 
-Tags: 6.8.3-jdk8, 6.8.3-jdk8-hotspot, 6.8-jdk8, 6.8-jdk8-hotspot, jdk8, jdk8-hotspot, 6.8.3-jdk, 6.8.3-jdk-hotspot, 6.8-jdk, 6.8-jdk-hotspot, jdk, jdk-hotspot, 6.8.3, 6.8.3-hotspot, 6.8, 6.8-hotspot, latest, hotspot
+Tags: 7.0.0-jdk8, 7.0.0-jdk8-hotspot, 7.0-jdk8, 7.0-jdk8-hotspot, jdk8, jdk8-hotspot, 7.0.0-jdk, 7.0.0-jdk-hotspot, 7.0-jdk, 7.0-jdk-hotspot, jdk, jdk-hotspot, 7.0.0, 7.0.0-hotspot, 7.0, 7.0-hotspot, latest, hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
```

Comment:
"Naughty" says you should remove the two ARM architectures from here (and indeed [the base image doesn't support those architectures](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/8-jdk-hotspot/images/sha256-b0123b7758d74f5ff3b22c4d4fe27df2dac92d02d99af9d3c2461bea9490ebf2?context=explore))
```suggestion
Architectures: amd64, ppc64le, s390x
```

Reply:
This is a constant struggle, and I don't know what to do about it.  AdoptOpenJDK folks keep adding and removing those architectures, depending on the timing of those binaries being released in relation to when they update the Docker images. So it's always a coin toss as to whether my PR will fail the check or not.